3741/1/lib/check_html.rb

```diff
@@ -0,0 +1,11 @@
+require 'json'
+# check url validate and return base html or nil object
+class CheckHtml
+  def call(gem_name)
```

Comment:
don't name it **call** - unless you have U absolutely need it: (exp. rack convention)